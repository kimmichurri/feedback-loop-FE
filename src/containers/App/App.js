import React from 'react';
import './App.scss';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <p>
          Feedback Loop
        </p>
      </header>
    </div>
  );
}

export default App;
