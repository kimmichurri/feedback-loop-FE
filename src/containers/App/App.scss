.App {
  text-align: center;
}

.App-header {
  color: black;
}

.route-container {
    background: linear-gradient( rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.3) ), url('./assets/laptops.jpg');
    background-position: center;
    background-size: cover;
    background-repeat: no-repeat;
    height: 85vh;
    padding-top: 50px;
}
