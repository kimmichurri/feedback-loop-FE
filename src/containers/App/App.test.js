import React from 'react';
import { App, mapStateToProps } from './App';
import { shallow } from 'enzyme';

describe('App', () => {
  let wrapper;
  let mockBool;
  let mockError;

  beforeEach(() => {
    mockBool = false
    mockError = "Something went wrong"
    wrapper = shallow(
      <App
         isLoading={mockBool}
         error={mockError}
      />
    )
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });
})