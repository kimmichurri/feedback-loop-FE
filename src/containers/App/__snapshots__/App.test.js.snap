// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<div
  className="App"
>
  <div>
    <Header />
  </div>
  <div
    className="route-container"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/register"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/dashboard"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/new-survey"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/student-dashboard"
      />
      <Route
        component={[Function]}
        path="/student-survey"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/recipients"
      />
      <Route
        component={[Function]}
      />
    </Switch>
  </div>
  <div>
    <Footer />
  </div>
</div>
`;
